vendors/espressif/boards/esp32/aws_tests/config_files/FreeRTOSIPConfig.h

```diff
@@ -230,9 +235,10 @@ extern uint32_t ulRand();
 
 /* The MTU is the maximum number of bytes the payload of a network frame can
  * contain.  For normal Ethernet V2 frames the maximum MTU is 1500.  Setting a
- * lower value can save RAM, depending on the buffer management scheme used.  If
- * ipconfigCAN_FRAGMENT_OUTGOING_PACKETS is 1 then (ipconfigNETWORK_MTU - 28) must
- * be divisible by 8. */
+ * lower value can save RAM.
+ * When the distance between Wi-Fi station and access point is long, or when
+ * the signal passes through walls, packets may get lost.  In that case it
+ * helps to decrease the MTU. */
```

Comment:
Some comment as above - this guidance doesn't seem very scientific to me.

Reply:
~~~c
/* When the distance between Wi-Fi station and access point is long, or when
 * the signal passes through walls, packets may get lost.  In that case it
 * helps to decrease the MTU. */
~~~

> Some comment as above - this guidance doesn't seem very scientific to me.

I admit that I am not a Wi-Fi specialist.
But what I experienced it that there is a relation between the configured MTU and the number of dropped packets. This only occurs when the Wi-Fi signal is weak,
Do you have a suggestion of how to write this in proper English?

> I'm nervous about adding this recommendation here unless we accompany
> it with an explanation of why this is the case.

I wouldn't go into the theory, just mention the observation. For me it took a long time debugging before it occurred to me that the Wi-Fi signal was weak.
Hein

Reply:
We should remove the comments on lines 239-241 – a compile time configuration isn't the right place to adjust for an environmental condition like signal strength, at least for most applications.  Also, since we're not sure of the exact mechanism, it would be better not comment on it until we know for sure what's happening.